detectron2_detector/README.md

```diff
@@ -0,0 +1,16 @@
+# detectron2_detector
+
+This package implements a 3D object detector with [Mask RCNN](https://arxiv.org/pdf/1703.06870.pdf) ([detectron2](https://github.com/facebookresearch/detectron2) implementation) and basic pointcloud processing. 
+We assume ordered pointcloud aligned with pixels is available. We use Mask RCNN to produce masks for objects, then use these masks to extract points belong to the objects and estimate position and size. 
+
+### Parameters
+| parameters       | Meaning        | Default |
+| ---------------- | ------------- | ------- |
+| detectron_config_file | config file to load detectron model, <br> check out [model zoo](https://github.com/facebookresearch/detectron2/blob/master/MODEL_ZOO.md) | COCO-InstanceSegmentation/mask_rcnn_R_50_FPN_3x.yaml |
+| detectron_score_thresh | min score to be considered as an object | 0.8 |
+| pointcloud2_topic | ros topic for pointcloud data | /realsense/camera/pointcloud |
```

Comment:
Default should be more general here and in the code

Reply:
Oh, also in the README, please  list that the pointcloud must be colored, that is a non-obvious and non-standard requirement